docs/development.md

```diff
@@ -107,9 +119,8 @@ function for performing config field validation, a function for determining
 which steps of an integration to ignore, and a list of steps that define how an
 integration should collect data.
 
-The SDK execution process will construct this object based on an opinionated
-directory layout. Use the provided
-[Integration Template](/template/README.md#project-structure) to get started.
+Use the provided [Integration Template](/template/README.md#project-structure)
```

Comment:
Just a question: Do we have this as a github template or a generator for this yet? I didn't see any issues made for it and I'm wondering if we should track that, otherwise its just copying it over.

Reply:
I think we should create an `integration-template` repo and make it a GitHub Template project. 👍 Created an Issue: https://github.com/JupiterOne/sdk/issues/199